tests/database/helpers/events.ts

```diff
@@ -166,6 +189,7 @@ export function eventTestHelper(pathAndEvents, helperName?) {
     actualPathAndEvents.splice(actualPathAndEvents.length - initializationEvents, initializationEvents);
     initializationEvents = 0;
 
+    initResolve();
```

Comment:
Nit: This is more of a "resolveInit()", since initX sounds like it would initialize X.